### ISSUE TITLE
Questions in datasets/plane_predict_dataset.py & Where to get '$ROOT_FOLDER/scans/scene%04d_%02d/scene%04d_%02d.txt'

### ISSUE THREAD

**Issue by Darkgleeman** [yi-ming-qian/interplane#7]
Hi,
when I used predict_all.py to evaluate the result, surprisingly I found that no result has been returned. I was certain that I followed up your instructions to download the dateset of PlaneRCNN and merged your relation_data.zip,so I checked for a long time how it was going? 

Finally I found my question in 'datasets/plane_predict_dataset.py'. In the "__init__" of 'class PlaneDataset'. When loading '/ScanNet/Tasks/Benchmark/scannetv1_test.txt' in line 35, many of lines in this txt were not contained in the variable 'planenet_scene_ids_val' in line 33 and even if some of them really existed, I sill stuck in the condition "if not os.path.exists(scenePath + '/' + scene_id + '.txt') or not os.path.exists(scenePath + '/annotation/planes.npy'):" in line 44. Therefore no scene could be successfully loaded. Why could it lead such result?

Besides,after I ignored that part of code , I still failed due to the lack of '$ROOT_FOLDER/scans/scene%04d_%02d/scene%04d_%02d.txt'.However, I thought myself following your instructions strictly, so I wondered where to get those txts.

Thanks beforehand.

**Comment by yi-ming-qian** [comment]
Hello, did you download the planercnn dataset? Those txts should come from planercnn.

**Comment by Darkgleeman** [comment]
Emmmm, actually following the instructions I had downloaded the dateset of ScanNet dataset (v2),plane annotation(44GB) and meta_data.zip, and merged them together as Planercnn said, but didn't find any clue about those txt files? Did I forget to download something containing those txt files?

Besides, I would appreciate a lot if you could tell me what's the use of such txt files? Do they stand for camera intrinsics or something else?Great thanks!

**Comment by yi-ming-qian** [comment]
Hello,

That part of code is directly copied from planercnn: https://github.com/NVlabs/planercnn/blob/master/datasets/plane_dataset.py#L34

The purpose is to remove planenet test set from the planercnn training set, for a fair comparison in the original planercnn paper. 

I guess an easier way to debug is can you locate the file "/ScanNet/Tasks/Benchmark/scannetv1_*.txt" in your downloaded dataset? If it contains, then maybe you are not setting a correct data path. Hope this helps.

**Comment by Darkgleeman** [comment]
Alas, I did have "/ScanNet/Tasks/Benchmark/scannetv1_*.txt" under the "$ROOT_FOLDER/". 

So would you mind helping me check whether your 'scene%04d_%02d.txt' is in the path '$ROOT_FOLDER/scans/scene%04d_%02d/scene%04d_%02d.txt' for reference? I want to check what on earth I had missed......(Under '$ROOT_FOLDER/scans' I only have **_annotation  color  depth  frames  intrinsic  pose  valid_images.npy_** those contents) Thank you for your generosity!

**Comment by yi-ming-qian** [comment]
Yes, I do have: 'scene%04d_%02d.txt' is in the path '$ROOT_FOLDER/scans/scene%04d_%02d/, which is from planercnn, I believe. 

A typical 'scene%04d_%02d.txt' looks like: 
colorHeight = 968
colorWidth = 1296
depthHeight = 480
depthWidth = 640
fx_color = 1170.187988
fx_depth = 577.870605
fy_color = 1170.187988
fy_depth = 577.870605
mx_color = 647.750000
mx_depth = 319.500000
my_color = 483.750000
my_depth = 239.500000
numColorFrames = 3052
numDepthFrames = 3052
numIMUmeasurements = 6430
sceneType = Conference Room

**Comment by Darkgleeman** [comment]
Fine, I will continue to check the operations in Planercnn repo. Thanks a lot for your great help!
